### ISSUE TITLE
Try updated browse-everything gem

### ISSUE THREAD

**Issue by hortongn** [uclibs/ucrate#349]
### Descriptive summary

browse-everything 0.16.1 has been released:
https://github.com/samvera/browse-everything/releases

UCrate is currently using a forked and modified version of browse-everything 0.14.2.  Try updating to 0.16.1 and research what's new.  We want to make sure that box.com still works, but it would be nice if other providers like Google Drive or Dropbox would work for us again too.

**Comment by scherztc** [comment]
0.16.1 was still buggy.  I had permission problems with view templates.
Box:
Using 0.15.2 I was able to connect to Box and attach a file to ingest.  
When I download the file from the work it downloaded as download.xxx.

DropBox:


Google Drive:

**Comment by scherztc** [comment]
https://www.dropbox.com/developers/reference/oauth-guide

**Comment by scherztc** [comment]
Tried curly
This site can’t provide a secure connection
curly.libraries.uc.edu sent an invalid response.
ERR_SSL_PROTOCOL_ERROR

**Comment by scherztc** [comment]
To test DropBox and Google we need SSL enabled on curly.

**Comment by scherztc** [comment]
I was able to test DropBox and GoogleDrive from scholar-dev.

BrowseEverything is not ready for production.  I have tested the 0.15.1, 0.16.1, and master branch and they all either have template problems, parameter problems, or attachment problems.

James Griffin is the only one actively developing on it and he just recently put in a few commits to fix Google drive just 12 days ago.  These changes have not be merged into master.
